build/Makefile

```
@@ -58,6 +58,7 @@ KIND_CONTAINER_NAME=$(KIND_PROFILE)-control-plane
 
 # Game Server image to use while doing end-to-end tests
 GS_TEST_IMAGE ?= gcr.io/agones-images/udp-server:0.21
+GS_TEST_IMAGE_TCPUDP ?= gcr.io/agones-dev-286708/tcpudp-server:0.1
```

Nice workaround. Please don't forget to switch to 
```suggestion
GS_TEST_IMAGE_TCPUDP ?= gcr.io/agones-images/tcpudp-server:0.1
```
Once your test would be stable.